keepers/basic-keeper.ts

```diff
@@ -11,6 +11,7 @@ async function prepareTransaction(
   version: string,
   arweaveProxy: ArweaveProxy): Promise<Transaction> {
     if (prices.length === 0) {
+      //TODO: what exactly "prepareTransaction" does? it doesn't seem to "keep"/store anything on AR..
```

Comment:
As discussed on daily stand up, it prepares transaction and signs it to get the id (which is saved in the cache layer)